numpy/ma/extras.py

```diff
@@ -1303,6 +1352,14 @@ def in1d(ar1, ar2, assume_unique=False, invert=False):
     -----
     .. versionadded:: 1.4.0
 
+    Examples
+    --------
+    >>> import numpy.ma as ma
+    >>> ar1 = np.ma.array([0, 1, 2, 5, 0])
+    >>> ar2 = [0, 2]
+    >>> np.in1d(ar1, ar2)
```

Comment:
I think you should use `ma.in1d`. This example doesn't use `in1d` from the `ma` module.